### PR TITLE
#6098 support multilingual remoteurl

### PR DESCRIPTION
**Pull request by mariuszsienkiewicz**
For this issue: https://github.com/pkp/pkp-lib/issues/6098

### CODE REVIEW

File: controllers/grid/navigationMenus/form/PKPNavigationMenuItemsForm.inc.php [hunk 1]
```diff
@@ -219,8 +220,11 @@ function validate($callHooks = true) {
 					$this->addError('path', __('manager.navigationMenus.form.duplicatePath'));
 				}
 			} elseif ($this->getData('menuItemType') == NMI_TYPE_REMOTE_URL) {
-				if(!filter_var($this->getData('url'), FILTER_VALIDATE_URL)) {
-					$this->addError('url', __('manager.navigationMenus.form.customUrlError'));
+				$remoteUrls = $this->getData('remoteUrl');
+				foreach ($remoteUrls as $remoteUrl) {
+					if(!filter_var($remoteUrl, FILTER_VALIDATE_URL)) {
+						$this->addError('url', __('manager.navigationMenus.form.customUrlError'));
```
Comment (asmecher) [inline controllers/grid/navigationMenus/form/PKPNavigationMenuItemsForm.inc.php:226]: I believe `'url'` should change to `'remoteUrl'` here.

File: templates/controllers/grid/navigationMenus/remoteUrlNMIType.tpl [hunk 2]
```diff
@@ -7,7 +7,7 @@
  *
  * Custom remote Url NMI Type edit form part
  *}
-{fbvFormSection id="NMI_TYPE_REMOTE_URL" class="NMI_TYPE_CUSTOM_EDIT" title="manager.navigationMenus.form.url" for="url" list=true required="true"}
-	{fbvElement type="text" id="url" value=$url maxlength="255" required="true"}
+{fbvFormSection id="NMI_TYPE_REMOTE_URL" class="NMI_TYPE_CUSTOM_EDIT" title="manager.navigationMenus.form.url" for="remoteUrl" list=true required="true"}
+    {fbvElement type="text" multilingual=true id="remoteUrl" value=$remoteUrl maxlength="255" required="true"}
```
Comment (asmecher) [inline templates/controllers/grid/navigationMenus/remoteUrlNMIType.tpl:11]: Watch out for inadvertent changes to indentation; we use tabs throughout our code.

**Comment by asmecher** [comment]
@mariuszsienkiewicz, this is a very clean contribution, thanks! I've just made a couple of brief comments.

**Comment by mariuszsienkiewicz** [comment]
Thank you @asmecher for the review!
All the necessary changes have been made.

**Pull request merged**